packages/block-editor/src/components/off-canvas-editor/leaf-more-menu.js

```diff
@@ -19,11 +19,67 @@ const POPOVER_PROPS = {
 	variant: 'toolbar',
 };
 
+const BLOCKS_THAT_CAN_BE_CONVERTED_TO_SUBMENU = [
+	'core/navigation-link',
+	'core/navigation-submenu',
+];
+
+function AddSubmenuItem( { block, onClose } ) {
+	const { insertBlock, replaceBlock, replaceInnerBlocks } =
+		useDispatch( blockEditorStore );
+
+	const clientId = block.clientId;
+	const isDisabled =
+		BLOCKS_THAT_CAN_BE_CONVERTED_TO_SUBMENU.indexOf( block.name ) === -1;
```

Comment:
```suggestion
	const isDisabled =
		! BLOCKS_THAT_CAN_BE_CONVERTED_TO_SUBMENU.includes( block.name );
```

Shall we utilise `includes`?